components/event-publisher-proxy/pkg/options/options.go

```diff
@@ -4,13 +4,17 @@ import "flag"
 
 type Options struct {
 	MaxRequestSize int64
+	MetricsAddress string
 }
 
 func ParseArgs() *Options {
-	maxRequestSize := flag.Int64("maxRequestSize", 65536, "The maximum request size in bytes")
+	maxRequestSize := flag.Int64("max-request-Size", 65536, "The maximum request size in bytes.")
```

Comment:
Here should be "max-request-size" too, right?

Reply:
Done